packages/astro/src/core/errors/errors-data.ts

```diff
@@ -791,6 +791,30 @@ export const InvalidDynamicRoute = {
 	message: (route: string, invalidParam: string, received: string) =>
 		`The ${invalidParam} param for route ${route} is invalid. Received **${received}**.`,
 } satisfies ErrorData;
+/**
+ * @docs
+ * @see
+ * - [Images](https://docs.astro.build/en/guides/images/)
```

Comment:
```suggestion
 * - [Default Image Service](https://docs.astro.build/en/guides/images/#default-image-service)
 * - [Image Component](https://docs.astro.build/en/guides/images/#image--astroassets)
 * - [Image Services API](https://docs.astro.build/en/reference/image-service-reference/)
```
These will all work in v3 docs